tests/driver_mq3/main.c

```diff
@@ -55,7 +55,7 @@ int main(void)
 
         printf("RAW: %4i, per mille: %1i.%03i\n", raw, alc_a, alc_b);
 
-        xtimer_usleep(500 * 1000);
+        xtimer_usleep((uint32_t)500 * 1000);
     }
     return 0;
 }
```

Comment:
I think this change should be in a separated PR. BTW, you can use the constants on `timex.h`